linera-service/benches/transfers.rs

```diff
@@ -67,8 +67,10 @@ async fn setup_native_token_balances(
         .then(|idx| {
             let key_pair = if idx % 2 == 0 {
                 AccountSecretKey::Secp256k1(Secp256k1SecretKey::generate())
-            } else {
+            } else if idx % 3 == 0 {
                 AccountSecretKey::Ed25519(Ed25519SecretKey::generate())
+            } else {
+                AccountSecretKey::EvmSecp256k1(EvmSecretKey::generate())
             };
```

Comment:
Now secp256k1 has 50%, Ed25519 17% and EVM-secp256k1 33%.

Maybe simpler: `match idx % 3`